src/App.tsx

```diff
@@ -5,21 +5,23 @@ import { Routes, Route, Link } from 'react-router-dom'
 import { CollectionProvider } from 'hooks'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { ReactQueryDevtools } from 'react-query/devtools'
+import tw, { styled } from 'twin.macro'
+
 const queryClient = new QueryClient()
 
 const App = () => {
   return (
-    <div className="App">
-      <nav className="bg-gradient-to-r from-blue-300 to-purple-300 py-16">
-        <ul className="m-0 list-none">
-          <Link className="nav-button" to="/">
-            My Collection
+    <div>
+      <Nav>
+        <ul tw="m-0 list-none">
```

Comment:
I would just make everything styled components instead of mixing and matching with inline classes